microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -6,259 +6,405 @@ ms.author: msedgedevrel
 
-### Applying the new environment variable to running processes
+* Option 1: Manually install preview channels on machines.
 
-After setting an environment variable, the environment variable is applied to any new processes that are created.  The environment variable doesn't apply to processes which are already running.  To ensure that all processes use the new environment variable, you may need to restart Visual Studio, or log out of Windows and then log in again.
+* Option 2: Programmatically deploy preview channels via API endpoints.
 
+These options are explained below.
 
-<!-- 4. Group Policy ====================================================== -->
-## Using a group policy
 
-If you want to make your application use a Microsoft Edge preview channel by using a group policy, copy ADMX and ADML files to the `PolicyDefinitions` folder, as follows.
+<!-- ---------- -->
+###### Option 1: Manually install preview channels on machines
 
-1. Download the policy files from [Download and deploy Microsoft Edge for business](https://www.microsoft.com/edge/business/download).
+Insider channels can be manually installed (either manually or through a deployed custom script) through the following enterprise links:
 
-1. Copy the ADMX file into a Policy Definitions template folder, such as `C:\Windows\PolicyDefinitions`.
+| Channel | Link |
+| --- | --- |
+| Canary | [Download](https://go.microsoft.com/fwlink/?linkid=2084649&Channel=Canary&language=en) |
+| Dev | [Download](https://go.microsoft.com/fwlink/?linkid=2093291) |
+| Beta | [Download](https://go.microsoft.com/fwlink/?linkid=2093376) |
 
-1. Copy the ADML file into a matching locale folder within the `Policy Definitions` folder, such as a `C:\Windows\PolicyDefinitions\en-us`<!--keep /en-us here--> folder.
+Manual installation only needs to be done once per machine.  Prerelease channels are evergreen, so they will automatically get updated when newer versions are available.
 
-1. Open the **Local Group Policy Editor**.  To do this, in the Windows search bar, type "group policy", and then select **Edit group policy**.
 
-1. Expand **Local Computer Policy**, then **Computer Configuration** or **User Configuration**.  Then expand **Administrative Templates** > **Microsoft Edge WebView2**.
+<!-- ---------- -->
+###### Option 2: Programmatically deploy preview channels via API endpoints
 
-   ![Local Group Policy Editor dialog](./set-preview-channel-images/local-group-policy-editor.png)
+In your app's code, you can write your own custom logic to deploy the latest version of the preview channel of Microsoft Edge by doing the following:
 
-1. Select **Browser Executable Folder**.  The following screenshots apply to setting the **Browser Executable Folder**.  Alternatively, select **Release Channel Preference**, which uses similar dialogs.
+1. Call <!-- todo: all of? any of? one of? --> the following API endpoints.
```

Comment:
Call all 3 API endpoints, or just one, or some/selected ones?
@victorhuangwq @champnic

Reply:
Rendered articles for review: 
* **Test upcoming APIs and features** (update to use new API)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/set-preview-channel?branch=pr-en-us-3053
* **Prerelease testing** (new article)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/prerelease-testing?branch=pr-en-us-3053
* **Self-hosting** (new article)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/self-hosting?branch=pr-en-us-3053

For "Outdated" comments, to see latest Markdown/commits, use fresh Files Changed tab:
https://github.com/MicrosoftDocs/edge-developer/pull/3053/files

Reply:
I'm not sure. @vickiez ?

Reply:
@mikehoffms they should call the endpoint corresponding to the preview channel that they want to deploy

Reply:
```
###### Option 2: Programmatically deploy preview channels via API endpoints

In your app's code, write your own custom logic to deploy the latest version of a preview channel of Microsoft Edge as follows:

1. Call the following API endpoint that corresponds to the preview channel that you want to deploy:
```